safe-global/safe-react
# feat: warning element for threshold changes/delegate calls

## What it solves
Resolves #3471 and #3723

## How this PR fixes it
A new `AlertTooltipWarning` component was created and added to (unexpected) delegate calls, as well as confirmation policy changes.

Confirmation policy change descriptions have also been adjusted.

Note: this does **not** include a link to an article regarding delegate calls.

## How to test it
- Open (unexpected) delegate call transactions and observe the warning message.
- Open threshold changing owner alterations and observe the warning message.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/167087826-81cb10be-0c3a-48ee-a620-92484566c17b.png)
![image](https://user-images.githubusercontent.com/20442784/167087849-c259adfc-689b-46eb-a795-98f91f390fb5.png)

(Unexpected) delegate call: `rin:0x73a7AA145338587f7aB7f63c06d187C85dF4727e/transactions/multisig_0x73a7AA145338587f7aB7f63c06d187C85dF4727e_0xba28109747222d6cfb7f0fb75f03d49957e343674bc116162b038e244dbcc6d6`
![image](https://user-images.githubusercontent.com/20442784/167082349-929824c5-3d36-40c8-be4e-2e805f7c6c12.png)
![image](https://user-images.githubusercontent.com/20442784/167082420-7b4bd433-e371-4b83-b4f9-1420faeaaadf.png)

Threshold alteration: (`rin:0xAb5e3288640396C3988af5a820510682f3C58adF/transactions/multisig_0xAb5e3288640396C3988af5a820510682f3C58adF_0x3a2f4ef3358d80bb776efabc9e29757441fe2fdba2dc2ead688ee9146a280def`)
![image](https://user-images.githubusercontent.com/20442784/167082153-de754b75-c7da-4b79-abf6-5c0629b5fe8a.png)
![image](https://user-images.githubusercontent.com/20442784/167082096-bf43955c-7cca-44a7-a3cb-69eb07a88591.png)
